apps/cdn/src/cache/key.rs

```diff
@@ -0,0 +1,235 @@
+use std::{fmt::Display, str::FromStr};
+
+use shared::database::{badge::BadgeId, emote::EmoteId, user::UserId};
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum CacheKey {
+	Badge {
+		id: BadgeId,
+		file: BadgeFile,
```

Comment:
we should also have the format too

Reply:
Currently, the format for badges is always webp